specification/ai/Azure.AI.Projects/agents/run_steps/models.tsp

```diff
@@ -265,7 +265,7 @@ model RunStepDeltaFileSearchToolCall extends RunStepDeltaToolCall {
 
   /** Reserved for future use. */
   @encodedName("application/json", "file_search")
-  fileSearch?: Record<string>;
+  fileSearch?: RunStepFileSearchToolCallResults;
```

Comment:
Is ` RunStepFileSearchToolCallResults` already defined somewhere?

Reply:
Yes, it was already used for the non-streaming response.